Seungsu/app.js

```diff
@@ -35,11 +37,23 @@ app.get("/ping", (req, res) => {
   res.json({ message: "pong" });
 });
 
-const server = http.createServer(app);
-const PORT = process.env.PORT;
+app.post("/signup", async (req, res, next) => {
+  const { name, email, password, profileImage } = req.body;
+  await myDataSource.query(
+    `INSERT INTO users(
+      name,
+      email,
+      password,
+      profile_image
+    ) VALUES (?, ?, ?, ?);
+    `,
+    [name, email, password, profileImage]
+  );
+  res.status(200).json({ message: "userCreated" });
```

Comment:
```suggestion
  res.status(201).json({ message: "userCreated" });
```
레코드 생성에 가장 적합한 상태코드는 201입니다. 숙지하셨다가 다음부터 적용해주세요~!

Reply:
네 알겠습니다 !